tests/suites/test_suite_camellia.function

```diff
@@ -7,6 +7,171 @@
  * END_DEPENDENCIES
  */
 
+/* BEGIN_CASE depends_on:MBEDTLS_CHECK_PARAMS:!MBEDTLS_PARAM_FAILED_ALT */
+void camellia_invalid_param( )
+{
+    mbedtls_camellia_context ctx;
+    unsigned char buf[16] = { 0 };
+    size_t off;
+    ((void) off);
+
+    TEST_INVALID_PARAM( mbedtls_camellia_init( NULL ) );
+    TEST_VALID_PARAM( mbedtls_camellia_free( NULL ) );
+
+    TEST_INVALID_PARAM_RET( MBEDTLS_ERR_CAMELLIA_BAD_INPUT_DATA,
+                            mbedtls_camellia_setkey_enc( NULL,
+                                                         buf,
+                                                         128 ) );
```

Comment:
It may be beneficial to call this `valid_keybits`, and the `42` below the `invalid_keybits`. This will improve the test's clarity, but I don't find this necessary, so I only suggest that.

Reply:
I will follow your suggestion.